### PR TITLE
Pract.01: Add 04-06 | Raya Grigorova

### PR DESCRIPTION
**Pull request by rayagrigorova**
Raya Grigorova
SI 1 5MI0600167

### CODE REVIEW

File: Seminars/Sem.01/Pract.01/Task06.cpp [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	return sqrt(dx * dx + dy * dy);
+}
+
+int getQuadrant(const Point& p) {
+	// 1 or 4
+	if (p.x > 0) {
+		//x is +, y is +
+		if (p.y > 0) return 1;
+		//x is +, y is -
+		return 4;
```
Comment (GeorgiTerziev02) [inline Seminars/Sem.01/Pract.01/Task06.cpp:39]: Use ternary operator

File: Seminars/Sem.01/Pract.01/Task06.cpp [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+		return 3;
+	}
+}
+
+//This function will return -1 if the point is outside the circle,
+//0 if it's on the outline and 1 if it's on the inside of the circle 
+int posRelativeToCircle(const Point& p, const Circle& c) {
+	//The center of the cirlce is the origin
+	double dist = getDistanceToOrigin(p);
+	double epsilon = 0.00001;
```
Comment (GeorgiTerziev02) [inline Seminars/Sem.01/Pract.01/Task06.cpp:54]: This should be gloabl variable

File: Seminars/Sem.01/Pract.01/Task04.cpp [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+const short MAX_NAME_LEN = 30;
+
+struct Student {
+    char name[MAX_NAME_LEN + 1];
+    unsigned int FN;
+};
+
+void initStudent(Student& student, const char name[], unsigned int FN) {
+    strcpy_s(student.name, MAX_NAME_LEN, name);
```
Comment (GeorgiTerziev02) [inline Seminars/Sem.01/Pract.01/Task04.cpp:11]: Write strcpy yourself

**Review by GeorgiTerziev02 — APPROVED** [review]
Good job. Check the comments and write str_cpy yourself!

File: Seminars/Sem.01/Pract.01/Task06.cpp [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+double getDistanceBetweenPoints(const Point& p1, const Point& p2) {
+	double dx = p1.x - p2.x;
+	double dy = p1.y - p2.y;
+
+	return sqrt(dx * dx + dy * dy);
+}
+
+int getQuadrant(const Point& p) {
+	// 1 or 4
+	if (p.x > 0) {
```
Comment (Justsvetoslavov) [inline Seminars/Sem.01/Pract.01/Task06.cpp:35]: Missing edge case if we are on X axis and/or Y-axis

**Review by Justsvetoslavov — APPROVED** [review]

**Comment by Justsvetoslavov** [comment]
Cherry-picked Task04 + Task06(although should be addressed)

**Pull request closed without merging**